### ISSUE TITLE
[BUG] <html> element does not have a [lang] attribute

### ISSUE THREAD

**Issue by squashfold** [squashfold/mdx-ts-nextjs-starter#14]
**Describe the bug**

If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly. [Learn more about the lang attribute](https://dequeuniversity.com/rules/axe/4.6/html-has-lang).

**To Reproduce**
1. Go to https://mdx-ts-nextjs-starter.vercel.app/
2. Run lighthouse test
3. See error

**Expected behavior**
lang attribute is set
